code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -292,6 +304,7 @@
 /datum/tarot/the_high_priestess
 	name = "II - The High Priestess"
 	desc = "Mother is watching you."
+	extended_desc = "alerts Bubblegum to the user, allowing them to rend them, dealing heavy damage, and imobilizing them."
```

Comment:
This feels a little weirdly worded, I'm assuming from watching Jimmy that this causes Bubblegum to attack other people unless yur the only one on the screen, yeh?

Reply:
yeah the normal card directly attacks the user of the card, not the ones around them, thats the reverse version